src/peer/repl/index.js

```diff
@@ -15,16 +28,18 @@ require('yargs')
       // this is the only way to get useful config in subcommands
       'config': true,
       'global': true,
-      'default': home + 'config.json'
+      'default': path.join(home, 'config.json')
     },
     'identityPath': {
       'global': true,
-      'default': home + 'identity.node'
+      'default': path.join(home, 'identity.node')
     },
     'remotePeer': {
       'global': true,
       'type': 'string',
       'describe': 'The remote peer to pair with (multiaddress)'
     }
   })
+  .coerce('config', filePath => ensureDirExists(filePath, 'config file'))
```

Comment:
you don't want to just put this into the `config` key above as `coerce : { filepath: ... `?